### DISCUSSION TITLE
复旦大学

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#57] [University]
> 来自群友

### 食
食堂普遍还算可口，除去本部旦苑食堂，在里面吃到过有蛋壳的番茄炒蛋，生的土豆块等等。
食堂刷校园卡消费。
吃腻了可以去附近的小餐馆或者五角场商圈。
学校内有多家教育超市，物价和外面的便利店没啥区别。

### 住
宿舍区有东区，南区，北区，本部。理工科（包括经济类，除去数学类）在南区宿舍，书桌与床分离，有上下铺。
南区部分宿舍有楼层内公用浴室和厕所，每层楼两个小隔间浴室，三个小隔间蹲厕。没有楼层内浴室的需要去附近的澡堂。一般的热水开放时间为 7-9 时，16-24 时。刷校园卡计费。
宿舍楼下有公用洗衣机，扫码使用，微信付费，不保证会不会越洗越脏。
宿舍楼下有开水房，只有滚烫的开水，刷校园卡计费，0.08RMB/30s。
教室和宿舍都有空调，宿舍空调按年付租赁费（本质是租遥控器）。
可以点外卖，送到就近的校门口自取，但要及时取，不然很可能被偷，校园开放时可以送到楼下。校内有多个菜鸟驿站，快递可以直接到菜鸟驿站取。

### 行
目前门禁非常严格，需要刷校园卡过门禁。
从正门步行 15 分钟即达五角场商业中心，但学校附近地铁站都有一定距离，目前正在修建校名地铁站。
校内可以骑行电瓶车，但需要到保卫处挂牌，宿舍内不可以给电瓶充电。
共享单车有哈罗和美团，高峰期间比较难找到，通常数量比较充裕。
医学类大二以后会搬到枫林校区，大一不要购置太多大型器具。枫林校区的各方面条件都比邯郸校区高好几个 level。
开学时会给学生发放农业银行储蓄卡，但需要自己去网点激活，未来的补贴和奖学金等都是打到这张卡上，不激活无法领取。

### 学
无早晚自习，无宵禁，无熄灯，有通宵教室。
是本人目前已知的唯一一所期中退课要收钱的学校，每学分 130 元。
如果当前学期有选修体育课（根据规定，毕业需要修四个学期的体育课），则需要刷锻。分为晨跑，必锻，选锻。一学期需要完成晨跑 12 次，下午可以刷 800米（6 次）、立定跳远（2 次）、引体向上/仰卧起坐（2 次）、任意选锻（6 次），晚上可刷选锻。选锻可以向体育部借器材练习。整体来说刷锻比较水，目前没有引进高校体育，全部都是老师盯着你刷。
寒假五周左右，暑假九周左右，七月有暑期课程，可以视情况选修。

**Comment by Hecate2** [comment]
复旦每门课最多只允许30%人得到A或A-成绩，分别对应绩点4.0和3.7。这会让出国时的纸面绩点不太好看。
